admin/class-boldgrid-backup-admin-auto-updates.php

```diff
@@ -77,7 +77,13 @@ public function set_settings() {
 			$this->settings = $boldgrid_backup_settings['auto_update'];
 		} else {
 			$this->settings = array(
-				'days' => 0,
+				'days'    => 0,
```

Comment:
For this method... we have [a method for getting the settings](https://github.com/BoldGrid/boldgrid-backup/blob/master/admin/class-boldgrid-backup-admin-settings.php#L98-L108). We also have [a way for getting one setting](https://github.com/BoldGrid/boldgrid-backup/blob/master/admin/class-boldgrid-backup-admin-settings.php#L92). What do you think about adding these defaults to that method, and simply call...

```php
$core           = apply_filters( 'boldgrid_backup_get_core', null );
$this->settings = $core->get_setting( 'auto_update' );
```

Reply:
What you are saying makes sense. I just like to avoid adding code to the Boldgrid_Backup_Admin_Core as much as possible since it is already so big as it is.

Reply:
It wouldn't be adding code to Core, but instead to the settings class :) The methods already exist, and they're already setup to set defaults, so you're just adding more defaults. It's not like you're adding new methods / etc.

Reply:
Sorry. I realized that after I submitted that comment. I hoped that I had deleted it before you saw it :D. I tried to do this like you had suggested, but the `apply_filters( 'boldgrid_backup_get_core', null );` did not work as extected. I ended up with this error.

![image](https://user-images.githubusercontent.com/49331357/88072678-5525a800-cb43-11ea-8c8e-49cbb443e11d.png)

Reply:
That is likely why I never used that method to begin with.